File: moneymanager-ui/src/main/java/com/tink/moneymanagerui/theme/TinkErrorSnackbarTheme.java
package com.tink.moneymanagerui.theme;

import android.content.Context;
import com.tink.moneymanagerui.R;
import se.tink.commons.extensions.ContextUtils;

public class TinkErrorSnackbarTheme extends TinkDefaultSnackbarTheme {

	Context context;

	public TinkErrorSnackbarTheme(Context context) {
		super(context);
		this.context = context;
	}

	@Override
	public int getBackgroundColor() {
		return ContextUtils.getColorFromAttr(context, R.attr.tink_warningColor);
	}
}

File: moneymanager-ui/src/main/java/com/tink/moneymanagerui/theme/TinkErrorSnackbarTheme.kt
package com.tink.moneymanagerui.theme

import android.content.Context
import com.tink.moneymanagerui.R
import se.tink.commons.extensions.getColorFromAttr

class TinkErrorSnackbarTheme(var context: Context) : TinkDefaultSnackbarTheme(context) {
    override val backgroundColor: Int
        get() = context.getColorFromAttr(R.attr.tink_warningColor)
}